<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoldersWidget</class>
 <widget class="QWidget" name="FoldersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>临时目录（固态硬盘）</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTreeWidget" name="treeSSD">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>目录</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>已使用</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>剩余</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>总容量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>使用情况</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAddSSDFolder">
          <property name="text">
           <string>添加目录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRemoveSSDFolder">
          <property name="text">
           <string>删除目录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>最终目录（机械硬盘）</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTreeWidget" name="treeHDD">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>挖矿</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>目录</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>已使用</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>剩余</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>总容量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>使用情况</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAddHDDFolder">
          <property name="text">
           <string>添加目录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRemoveHDDFolder">
          <property name="text">
           <string>删除目录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
